src/Database/IdentifierQuoter.php

```diff
@@ -32,7 +32,7 @@ class IdentifierQuoter
      *
      * @var \Cake\Database\Driver
      */
-    protected $_driver;
+    protected Driver $_driver;
```

Comment:
Should this `DriverInterface`?

Reply:
Looks like it can be. We really should rename Driver.